### PR TITLE
Add e2e-metal-ipi-serial into origin repo

### PR DESCRIPTION
**Pull request by ardaguclu**
New tests are being added into serial suite for metal IPI. This
PR adds this workflow as optional to test the new tests in PRs.

### CODE REVIEW

**Comment by ardaguclu** [comment]
@stbenjam could you please have a look at this new job in origin, it is optional and we'll use it when new serial tests are added. Thanks.

File: ci-operator/config/openshift/origin/openshift-origin-master.yaml [hunk 1]
```diff
@@ -267,6 +267,11 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e-virtualmedia
+- as: e2e-metal-ipi-serial-ipv4
```
Comment (stbenjam) [inline ci-operator/config/openshift/origin/openshift-origin-master.yaml:270]: ipv4 is assumed to be the default, it shouldn't be part of the test name to be consistent with other platforms

Comment (ardaguclu) [reply ci-operator/config/openshift/origin/openshift-origin-master.yaml]: Thanks @stbenjam, you are right. I changed that way.

**Pull request merged**